src/runtime-rs/config/configuration-dragonball.toml.in

```diff
@@ -207,8 +207,6 @@ container_pipe_size=@PIPESIZE@
 internetworking_model="@DEFNETWORKMODEL_DB@"
 
 name="@RUNTIMENAME@"
```

Comment:
According to your commit, I think the `runtime name` is also redundant.

Reply:
Yes, please could you clarify this point. If users can configure different runtime types, they need to know what the types are and what the types mean.

The `runtime.name=` is not present in the golang config files so either we accept this is a rust-runtime specific option, or make it even clearer by changing the name of the table from `[runtime]` to `[runtime.@RUNTIMENAME@]`.